modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountSupplierRole.testcase

```diff
@@ -15,6 +15,58 @@ definition {
 		CommerceConfiguration.commerceTearDown();
 	}
 
+	@description = "This is a test case for COMMERCE-11825. Admin can verify that supplier account Id is right even after changing the catalog's name"
```

Comment:
Admin can verify that changing the catalog's name does not affect the supplier account Id and it remains the same.